packages/app/src/debug/DebugRunNavigation.cy.tsx

```diff
@@ -205,6 +211,40 @@ describe('<DebugRunNavigation />', () => {
     })
   })
 
+  it('displays the limit message when the number of total runs is exactly 100', () => {
+    const latest = createRun({
+      runNumber: 3,
+      status: 'RUNNING',
+      sha: 'sha-123',
+      summary: 'add new feature with a really long commit message to see what happens',
+      completedInstanceCount: 5,
+      totalInstanceCount: 12,
+    })
+
+    const other1 = createRun({ runNumber: 1, status: 'PASSED', sha: 'sha-456', summary: 'Update code' })
+    const other2 = createRun({ runNumber: 2, status: 'PASSED', sha: 'sha-456', summary: 'Update code' })
+
+    const allRuns: any = []
```

Comment:
Could be more type safe!

```suggestion
    const allRuns: Array<typeof createRun> = []
```

Reply:
I added some elements in when I initialized the array so now it's implicitly typed